src/systems/hydrodynamics/Hydrodynamics.cc

```diff
@@ -350,12 +354,23 @@ void Hydrodynamics::Configure(
       prefix += "Dot";
       prefix += snameConventionVel[j];
       this->dataPtr->Ma(i, j) = SdfParamDouble(_sdf, prefix, 0);
+      if (std::abs(this->dataPtr->Ma(i, j)) > 1e-6)
+      {
+        added_mass_specified = true;
```

Comment:
Considering this is in a `for` loop, I think you want to change it to:

`addedMassSpecified = addedMassSpecified && (std::abs(this->dataPtr->Ma(i, j)) > 1e-6)`

You can then remove the `if` statement.